Services/CheckRightsService/Database/Migrations/20200828092725_AddRightDbUser.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.CheckRightsService.Database.Migrations
+{
+    public partial class AddRightDbUser : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(
+                table: "Rights",
+                keyColumn: "Id",
+                keyValue: 1);
+
+            migrationBuilder.DeleteData(
+                table: "Rights",
+                keyColumn: "Id",
+                keyValue: 2);
+
+            migrationBuilder.CreateTable(
+                name: "DbRightUser",
+                columns: table => new
+                {
+                    RightId = table.Column<int>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_DbRightUser", x => new { x.RightId, x.UserId });
+                    table.ForeignKey(
+                        name: "FK_DbRightUser_Rights_RightId",
+                        column: x => x.RightId,
+                        principalTable: "Rights",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+        }
```

Comment:
???

Reply:
So that dbContext query works properly.

Reply:
Anyway, I removed "delete data" sections.